docs/tutorial/population_dynamics.rst

```diff
@@ -278,6 +278,43 @@ These information criteria values weigh the log likelihood against the model com
 model selection. In general, the model with the lowest value is optimal. We can see that both values are lower for the double
 exponential model, but only slightly so it is not strong evidence to choose the more complex model.
 
+.. _pop_std_err:
+
+Standard errors
+^^^^^^^^^^^^^^^
+
+Obtaining lifetime estimates is important, but it is equally important to know how reliable these estimates are.
+One way to gauge the reliability of the estimates is to calculate the asymptotic standard errors of the parameters.
+We can find these in the `err_amplitudes` and `err_lifetimes` properties.
```

Comment:
I don’t think you need to explain that knowing the reliability of an estimate is important. How about rephrasing the above 3 lines as:

After the optimal model for the data has been selected using the methods above, the reliability of the parameter estimates can be determined. Pylake provides several methods to determine both the confidence intervals and the standard errors of the parameters. 
For large datasets, the standard error under `err_amplitudes` and `err_lifetimes` can be used. Note that these are only asymptotically correct:

Reply:
> I don’t think you need to explain that knowing the reliability of an estimate is important.

Haha, yes, fair point! 😅 

I will swap it out for your section!